setup.py

```diff
@@ -49,9 +49,11 @@
     ],
     'postgresql': [
         'invenio-db[postgresql,versioning]>=1.0.0a9',
+        'psycopg2',
```

Comment:
@david-caro can you reproduce the problem as a test-case for `requirements-builder` [1]?

--
[1] https://github.com/inveniosoftware/requirements-builder